app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/Model/Category.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.HumanLearning2021.HumanLearningApp.Model
+
+
+/**
+ * Class representing a category, which can be used to classify images
+ *
+ * @param name the name of the category
+ */
+class Category(val name: String) {
+
+    override fun equals(other: Any?): Boolean {
+        return other is Category && other.name.equals(name, ignoreCase = true)
```

Comment:
It might be desirable to do the [Odersky, Spoon, Venners canEqual](https://www.artima.com/articles/how-to-write-an-equality-method-in-java) here, given that I intend to inherit this class in the Firestore module.